storage/bootstrap/bootstrapper/commitlog/source.go

```diff
@@ -116,7 +116,7 @@ func (s *commitLogSource) Read(
 	unmerged := make([]encodersAndRanges, numShards)
 	for shard := range shardsTimeRanges {
 		unmerged[shard] = encodersAndRanges{
-			encodersBySeries: make(map[ident.Hash]encodersByTime),
+			encodersBySeries: make(map[uint64]encodersByTime),
```

Comment:
if this map starts becoming a bottleneck, https://github.com/brentp/intintmap might be interesting to play with.

Reply:
Could we use this for the metadata lookup? map unique series index to index in a slice where I append the metadata as they appear? Probably not gonna test it right now, but will investigate when I come back to optimizing this

Reply:
👍 makes sense